### ISSUE TITLE
JSON schema validation fails on build agent without internet connection

### ISSUE THREAD

**Issue by nil4** [CycloneDX/cyclonedx-maven-plugin#100]
We have observed a regression from `cyclonedx-maven-plugin:2.2.0` to `cyclonedx-maven-plugin:2.4.0` on a build agent with no internet connection (mandated by regulatory requirements). 

The command line we use:

```
mvn -f pom.xml "-DSPDXParser.OnlyUseLocalLicenses=true" org.cyclonedx:cyclonedx-maven-plugin:2.2.0:makeAggregateBom
```

This works offline with 2.2.0, and produces the `bom.xml` file as expected.

But after updating to `org.cyclonedx:cyclonedx-maven-plugin:2.4.0:makeAggregateBom`, the build fails, apparently because it generates both XML and a JSON BOMs, and then attempts to validate the JSON BOM expecting online access:

```
[INFO] CycloneDX: Creating BOM
[INFO] CycloneDX: Writing BOM (XML): F:\1040\s\target\bom.xml
[INFO] CycloneDX: Validating BOM (XML): F:\1040\s\target\bom.xml
[INFO] CycloneDX: Writing BOM (JSON): F:\1040\s\target\bom.json
[INFO] CycloneDX: Validating BOM (JSON): F:\1040\s\target\bom.json
[ERROR] Failed to load json schema!
java.net.UnknownHostException: cyclonedx.org
```

Is it possible to configure the plugin to either (a) not generate the JSON BOM at all (we only need XML), or (b) not attempt to validate the JSON schema, or (c) not require online access to validate? Any guidance would be greatly appreciated.

<details>
<summary>Maven log including relevant stack traces</summary>
<pre><code>mvn -f F:\1040\s\pom.xml "-DSPDXParser.OnlyUseLocalLicenses=true" org.cyclonedx:cyclonedx-maven-plugin:2.4.0:makeAggregateBom

[INFO] CycloneDX: Creating BOM
[INFO] CycloneDX: Writing BOM (XML): F:\1040\s\target\bom.xml
[INFO] CycloneDX: Validating BOM (XML): F:\1040\s\target\bom.xml
[INFO] CycloneDX: Writing BOM (JSON): F:\1040\s\target\bom.json
[INFO] CycloneDX: Validating BOM (JSON): F:\1040\s\target\bom.json
[ERROR] Failed to load json schema!
java.net.UnknownHostException: cyclonedx.org
    at java.net.AbstractPlainSocketImpl.connect (AbstractPlainSocketImpl.java:220)
    at java.net.Socket.connect (Socket.java:609)
    at java.net.Socket.connect (Socket.java:558)
    at sun.net.NetworkClient.doConnect (NetworkClient.java:182)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:474)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:569)
    at sun.net.www.http.HttpClient.<init> (HttpClient.java:242)
    at sun.net.www.http.HttpClient.New (HttpClient.java:341)
    at sun.net.www.http.HttpClient.New (HttpClient.java:362)
    at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient (HttpURLConnection.java:1253)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect0 (HttpURLConnection.java:1187)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect (HttpURLConnection.java:1081)
    at sun.net.www.protocol.http.HttpURLConnection.connect (HttpURLConnection.java:1015)
    at sun.net.www.protocol.http.HttpURLConnection.getInputStream0 (HttpURLConnection.java:1592)
    at sun.net.www.protocol.http.HttpURLConnection.getInputStream (HttpURLConnection.java:1520)
    at com.networknt.schema.uri.URLFetcher.openConnectionCheckRedirects (URLFetcher.java:57)
    at com.networknt.schema.uri.URLFetcher.fetch (URLFetcher.java:43)
    at com.networknt.schema.uri.URISchemeFetcher.fetch (URISchemeFetcher.java:50)
    at com.networknt.schema.JsonSchemaFactory.getSchema (JsonSchemaFactory.java:337)
    at com.networknt.schema.RefValidator.getRefSchema (RefValidator.java:83)
    at com.networknt.schema.RefValidator.<init> (RefValidator.java:46)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:490)
    at com.networknt.schema.ValidatorTypeCode.newValidator (ValidatorTypeCode.java:131)
    at com.networknt.schema.JsonMetaSchema.newValidator (JsonMetaSchema.java:351)
    at com.networknt.schema.ValidationContext.newValidator (ValidationContext.java:54)
    at com.networknt.schema.JsonSchema.read (JsonSchema.java:206)
    at com.networknt.schema.JsonSchema.getValidators (JsonSchema.java:356)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:227)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.BaseJsonValidator.validate (BaseJsonValidator.java:96)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:234)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:219)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:124)
    at org.cyclonedx.parsers.JsonParser.isValid (JsonParser.java:267)
    at org.cyclonedx.maven.BaseCycloneDxMojo.createBom (BaseCycloneDxMojo.java:798)
    at org.cyclonedx.maven.BaseCycloneDxMojo.execute (BaseCycloneDxMojo.java:763)
    at org.cyclonedx.maven.CycloneDxAggregateMojo.execute (CycloneDxAggregateMojo.java:124)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR] Error:
java.lang.reflect.InvocationTargetException
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:490)
    at com.networknt.schema.ValidatorTypeCode.newValidator (ValidatorTypeCode.java:131)
    at com.networknt.schema.JsonMetaSchema.newValidator (JsonMetaSchema.java:351)
    at com.networknt.schema.ValidationContext.newValidator (ValidationContext.java:54)
    at com.networknt.schema.JsonSchema.read (JsonSchema.java:206)
    at com.networknt.schema.JsonSchema.getValidators (JsonSchema.java:356)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:227)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.BaseJsonValidator.validate (BaseJsonValidator.java:96)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:234)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:219)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:124)
    at org.cyclonedx.parsers.JsonParser.isValid (JsonParser.java:267)
    at org.cyclonedx.maven.BaseCycloneDxMojo.createBom (BaseCycloneDxMojo.java:798)
    at org.cyclonedx.maven.BaseCycloneDxMojo.execute (BaseCycloneDxMojo.java:763)
    at org.cyclonedx.maven.CycloneDxAggregateMojo.execute (CycloneDxAggregateMojo.java:124)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: com.networknt.schema.JsonSchemaException: java.net.UnknownHostException: cyclonedx.org
    at com.networknt.schema.JsonSchemaFactory.getSchema (JsonSchemaFactory.java:361)
    at com.networknt.schema.RefValidator.getRefSchema (RefValidator.java:83)
    at com.networknt.schema.RefValidator.<init> (RefValidator.java:46)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:490)
    at com.networknt.schema.ValidatorTypeCode.newValidator (ValidatorTypeCode.java:131)
    at com.networknt.schema.JsonMetaSchema.newValidator (JsonMetaSchema.java:351)
    at com.networknt.schema.ValidationContext.newValidator (ValidationContext.java:54)
    at com.networknt.schema.JsonSchema.read (JsonSchema.java:206)
    at com.networknt.schema.JsonSchema.getValidators (JsonSchema.java:356)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:227)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.BaseJsonValidator.validate (BaseJsonValidator.java:96)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:234)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:219)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:124)
    at org.cyclonedx.parsers.JsonParser.isValid (JsonParser.java:267)
    at org.cyclonedx.maven.BaseCycloneDxMojo.createBom (BaseCycloneDxMojo.java:798)
    at org.cyclonedx.maven.BaseCycloneDxMojo.execute (BaseCycloneDxMojo.java:763)
    at org.cyclonedx.maven.CycloneDxAggregateMojo.execute (CycloneDxAggregateMojo.java:124)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.net.UnknownHostException: cyclonedx.org
    at java.net.AbstractPlainSocketImpl.connect (AbstractPlainSocketImpl.java:220)
    at java.net.Socket.connect (Socket.java:609)
    at java.net.Socket.connect (Socket.java:558)
    at sun.net.NetworkClient.doConnect (NetworkClient.java:182)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:474)
    at sun.net.www.http.HttpClient.openServer (HttpClient.java:569)
    at sun.net.www.http.HttpClient.<init> (HttpClient.java:242)
    at sun.net.www.http.HttpClient.New (HttpClient.java:341)
    at sun.net.www.http.HttpClient.New (HttpClient.java:362)
    at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient (HttpURLConnection.java:1253)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect0 (HttpURLConnection.java:1187)
    at sun.net.www.protocol.http.HttpURLConnection.plainConnect (HttpURLConnection.java:1081)
    at sun.net.www.protocol.http.HttpURLConnection.connect (HttpURLConnection.java:1015)
    at sun.net.www.protocol.http.HttpURLConnection.getInputStream0 (HttpURLConnection.java:1592)
    at sun.net.www.protocol.http.HttpURLConnection.getInputStream (HttpURLConnection.java:1520)
    at com.networknt.schema.uri.URLFetcher.openConnectionCheckRedirects (URLFetcher.java:57)
    at com.networknt.schema.uri.URLFetcher.fetch (URLFetcher.java:43)
    at com.networknt.schema.uri.URISchemeFetcher.fetch (URISchemeFetcher.java:50)
    at com.networknt.schema.JsonSchemaFactory.getSchema (JsonSchemaFactory.java:337)
    at com.networknt.schema.RefValidator.getRefSchema (RefValidator.java:83)
    at com.networknt.schema.RefValidator.<init> (RefValidator.java:46)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
    at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:490)
    at com.networknt.schema.ValidatorTypeCode.newValidator (ValidatorTypeCode.java:131)
    at com.networknt.schema.JsonMetaSchema.newValidator (JsonMetaSchema.java:351)
    at com.networknt.schema.ValidationContext.newValidator (ValidationContext.java:54)
    at com.networknt.schema.JsonSchema.read (JsonSchema.java:206)
    at com.networknt.schema.JsonSchema.getValidators (JsonSchema.java:356)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:227)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.JsonSchemaRef.validate (JsonSchemaRef.java:49)
    at com.networknt.schema.RefValidator.validate (RefValidator.java:138)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.ItemsValidator.doValidate (ItemsValidator.java:93)
    at com.networknt.schema.ItemsValidator.validate (ItemsValidator.java:80)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.PropertiesValidator.validate (PropertiesValidator.java:70)
    at com.networknt.schema.JsonSchema.validate (JsonSchema.java:229)
    at com.networknt.schema.BaseJsonValidator.validate (BaseJsonValidator.java:96)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:234)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:219)
    at org.cyclonedx.parsers.JsonParser.validate (JsonParser.java:124)
    at org.cyclonedx.parsers.JsonParser.isValid (JsonParser.java:267)
    at org.cyclonedx.maven.BaseCycloneDxMojo.createBom (BaseCycloneDxMojo.java:798)
    at org.cyclonedx.maven.BaseCycloneDxMojo.execute (BaseCycloneDxMojo.java:763)
    at org.cyclonedx.maven.CycloneDxAggregateMojo.execute (CycloneDxAggregateMojo.java:124)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)</code></pre>
</details>

**Comment by stevespringett** [comment]
Duplicate of #94
Fixed in 2.4.1

**Comment by msymons** [comment]
@nil4, the plugin generates both JSON and XML by default as, the default configuration is:
```
<outputFormat>all</outputFormat>
```
See the readme.

Just tweak the configuration to get xml but not json:

```
<outputFormat>xml</outputFormat>
```

**Comment by nil4** [comment]
Confirmed fixed in 2.4.1 -- much appreciated!
